README.md

```diff
@@ -114,7 +118,7 @@ export AWS_SECRET_ACCESS_KEY=bar
 ```
 
 ### Add the SSH Key to the SSH Agent
-You will need to add the private key to your SSH agent: 
+Create your EC2 Key Pair with name "default" in the AWS Region you picked.  You will need to add the private key to your SSH agent: 
```

Comment:
I'm happy for a larger refactoring of getting the defaults to work. Perhaps the prereqs need to be revisited.